### PR TITLE
Open dialog and add credential with URL option

### PR DESCRIPTION
**Pull request by snaag**
- Open dialog, add credential when path from action=add
- #1394

### CODE REVIEW

File: src/components/backend-ai-credential-view.ts [hunk 1]
```diff
@@ -995,6 +996,16 @@ export default class BackendAICredentialView extends BackendAIPage {
     return Math.round(gigabyte * value);
   }
 
+  async _runAction() {
+    const action = location.search.split('action=')[1];
```
Comment (inureyes) [inline src/components/backend-ai-credential-view.ts]: @snaag This line will generate error when ?action GET parameter is missing.

image

**Review by inureyes — CHANGES_REQUESTED** [review]
@snaag `_runAction` generates an error when `action` GET parameter is missing.

File: src/components/backend-ai-credential-view.ts (hunk 1 above)
Comment (snaag) [reply src/components/backend-ai-credential-view.ts]: Oh right. Modified by adding code for checking.

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**